### PR TITLE
159965433 Add TravisCI configurations to .travis.yml

### PR DESCRIPTION
**Pull request by Quantum-35**
#### What does this PR do?
Creates configurations to make tests build successfully in Travis before submitting the coverage to coveralls
#### Description of tasks to be completed
This PR creates a test_database that will enable our tests to build successful and then submit the coverage to Coveralls for test coverage.
#### How should this be manually tested?
Once tests are written, they will build successfully 
#### Relevant PT story
[#159965433](https://www.pivotaltracker.com/story/show/159965433)

### CODE REVIEW

**Comment by DerKip** [comment]
considering it's now pointing to the developer branch, it's okay

**Comment by Mutuba** [comment]
I do not see an issue with it!

**Comment by Derrickkip** [comment]
Fix the failing checks

**Comment by Quantum-35** [comment]
@Derrickkip The checks are failing because there is no tests.

File: .gitignore [hunk 1]
```diff
@@ -2,7 +2,7 @@
 __pycache__/
 *.py[cod]
 *$py.class
-
+.vscode/
```
Comment (rethes) [inline .gitignore]: update with pycharm files

**Comment by ghost** [comment]
Update the gitignore with pycharm files

**Review by Teddykavooh — COMMENTED** [review]
Linting is commendable.

**Comment by Quantum-35** [comment]
What files @rethes

**Pull request closed without merging**